schema/settings-model.json

```diff
@@ -4,6 +4,12 @@
   "description": "Configuration for JupyterLite AI extension providers, models, and behavior",
   "type": "object",
   "properties": {
+    "useSecretsManager": {
+      "type": "boolean",
+      "title": "Use secrets manager",
+      "description": "Whether to use or not the secrets manager. If not, secrets will be stored in plain text in settings",
+      "default": true
```

Comment:
Wondering if this should default to `false`? 

Looks like by default the secret manager does not use a connector, so that means API keys are lost after refreshing the page and need to be entered again?

Reply:
> Probably we will need to document the usage of the Secret Manager a little bit more.
:+1: 

> Looks like by default the secret manager does not use a connector, so that means API keys are lost after refreshing the page and need to be entered again?

The default is using an "in memory" connector, so indeed it is lost after reloading the page.
But I wonder if this behavior isn't better than storing the API key in plain text by default.

Reply:
Right, maybe it makes more sense like this.

And it can always be disabled via the UI or the settings.